vunit/vhdl/verification_components/run.py

```diff
@@ -22,7 +22,7 @@ def encode(tb_cfg):
 
 
 def gen_wb_tests(obj, *args):
-    for dat_width, num_cycles, strobe_prob, ack_prob, stall_prob in product(*args):
+    for dat_width, num_cycles, strobe_prob, ack_prob, stall_prob, slave_inst in product(*args):
```

Comment:
I would use `slave_comb` instead of `slave_inst`, since my understanding is that the difference between the two cases is whether the slave is combinatorial or registered.

Reply:
The idea is to have possibility to skip wb slave VC instantiation and drive wishbone slave signals directly. It makes possible to test various corner cases - like here when specific set of acknowledge logic lead to crash on modelsim, but not ghdl. I thinks it's not feasible to add this corner cases reproducers to wishbone slave as it will obfuscate code. This is all about wb master VC bug.